src/VisualBoard.ts

```diff
@@ -1,18 +1,36 @@
 
-export function render(board: Array<Array<number>>) {
+export function buildGraphicalBoard(board: Array<Array<number>>): String {
     const alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
-    const lettresEntete = ' ' + alphabet.substr(0, board[0].length)
-    const entete = lettresEntete.split("").map(char => ` ${char} `)
-    const firstLine = entete.join("")
-
-    console.log(firstLine)
+    const headerLetter = ' ' + alphabet.substr(0, board[0].length)
```

Comment:
This is mixing up two things:

1. reducing the alphabet to get only the column names you want (in which case your variable could be named "columnLetters"
2. Adding a space at the beginning
Step 2 should be done below (line 6 or 8)